uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -648,12 +649,7 @@ private function createAbsencePeriod($startDate, $endDate) {
   }
 
   private function createAbsenceType() {
-    return AbsenceType::create([
-      'title' => 'Type ' . microtime(),
-      'color' => '#000000',
-      'default_entitlement' => 20,
-      'allow_request_cancelation' => 1,
-    ]);
+    return AbsenceTypeFabricator::fabricate();
```

Comment:
Sorry if I wasn't clear. My suggestion was to completely replace the helper method (`createAbsenceType()`) with the fabricator. Can you please do that here and in the other classes?